sdk/python/touca/_runner.py

```diff
@@ -389,12 +390,14 @@ def _run_workflow(options, workflow):
         stats.inc(status)
 
         if not errors and options.get("save-as-binary"):
-            Client.instance().save_binary(
-                os.path.join(casedir, "touca.bin"), [testcase]
-            )
+            full_path = os.path.join(casedir, "touca.bin")
+            Client.instance().save_binary(full_path, [testcase])
+            print(f"binary saved in {full_path}")
 
         if not errors and options.get("save-as-json"):
-            Client.instance().save_json(os.path.join(casedir, "touca.json"), [testcase])
+            full_path = os.path.join(casedir, "touca.json")
```

Comment:
This implementation prints the path for every test case that is executed. I don't think we want that given that a typical Touca test has tens or hundreds of testcases. I think it'd be nicer if we printed the path to the `results` directory, after all test cases are executed.